htdsa/client.py

```diff
@@ -59,7 +60,15 @@ def _uri(self, args=None):
 		return uri
 	
 	def _request(self, method, args, kwargs):
-		result = self.pool.request(method, self._uri(args), data=kwargs, **self.options)
+		uri = self._uri(args)
+		data = kwargs
+		if method == 'GET' and len(kwargs) > 0:
+			data = dict()
```

Comment:
Very minor code style thing, `dict()` may be quite readable, but it is actually slower than a dictionary literal like `{}`.  (Scoped name resolution + function call vs. bytecode constant load.)